jaxws-retail/service/pom.xml

```diff
@@ -47,7 +47,12 @@
             <artifactId>jboss-ejb-api_3.2_spec</artifactId>
             <scope>provided</scope>
         </dependency>
-        <!-- used to pull correct dependencies for jaxws-tools-maven-plugin  -->
+        <!-- The JAXWS Client BOM provides management dependencies compatible with the app server,
+        useful for building, testing or debugging JAXWS client applications. Import it to the app's pom.xml
+        dependencyManagement and then set dependencies on specific artifacts, or alternatively simply set
+        a single dependency on it for the app to depend on all the artifacts it manages, as done here.
+        In concrete here it's used to pull correct dependencies for jaxws-tools-maven-plugin.
+        -->
```

Comment:
In concrete here it's used to pull correct dependencies for jaxws-tools-maven-plugin.-->
In the following example, the BOM is used to pull the correct dependencies for jaxws-tools-maven-plugin.